cypress/e2e/buy-shirt.cy.ts

```diff
@@ -1,29 +1,41 @@
-describe("Buy a t-shirt", () => {
-  it("then the t-shirt should be bought", () => {
-    cy.visit("http://automationpractice.com/");
-    cy.get("#block_top_menu > ul > li:nth-child(3) > a").click();
-    cy.get(
-        "#center_column a.button.ajax_add_to_cart_button.btn.btn-default",
-    ).click();
-    cy.get("[style*='display: block;'] .button-container > a").click();
-    cy.get(".cart_navigation span").click();
-
-    cy.get("#email").type("aperdomobo@gmail.com");
-    cy.get("#passwd").type("WorkshopProtractor");
-
-    cy.get("[name=SubmitLogin]").click();
-    cy.get(".cart_navigation span").click();
+import {MenuContentPage} from "../page/index";
+import {ProductslistPage} from "../page/index";
+import {ShopingCartPage} from "../page/index";
+import {LoginPage} from "../page/index";
+import {AdressStepPage} from "../page/index";
+import {ShippingStepPage} from "../page/index";
+import {PaymentStepPage} from "../page/index";
 
-    cy.get("[name=cgv]").click();
-    cy.get(".cart_navigation span").click();
+const menuContentPage = new MenuContentPage();
+const productsListPage = new ProductslistPage();
+const shopingCartPage = new ShopingCartPage();
+const loginPage = new LoginPage();
+const adressStepPage = new AdressStepPage();
+const shippingStepPage = new ShippingStepPage();
+const paymentStepPage = new PaymentStepPage();
 
-    cy.get("[title='Pay by bank wire']").click();
-    cy.get(".cart_navigation span").click();
+describe("Buy a t-shirt", () => {
+  it("then should be bought a t-shirt", () => {
+    menuContentPage.visitMenuContentPage();
+    menuContentPage.goToTShirtMenu();
+    productsListPage.goToproductList();
+    productsListPage.goToproceedToCheckout();
+    shopingCartPage.goToproceedToCheckoutSummary();
+    loginPage.writeEmail("aperdomobo@gmail.com");
+    loginPage.writePassword("WorkshopProtractor");
+    loginPage.goToSubmit();
+    adressStepPage.goToproceedToCheckoutAdress();
+    shippingStepPage.getCheck();
```

Comment:
here, we can specify that we are accepting terms and conditions, son a better name can be: acceptTerms or checkTerms